tests/components/controlBar-test.js

```diff
@@ -371,6 +371,108 @@ describe('ControlBar', function () {
     expect(newVolume).toBeGreaterThan(-1);
   });
 
+  it('should display unmute volume icon when volume is set to a non-zero value', function() {
+    var mockController = {
```

Comment:
Can we use `baseMockController` here so that we don't repeat this each time?

Reply:
Yep, I'll make this change

Reply:
Done